main.lua

```diff
@@ -350,7 +367,9 @@ end
 
 function defaultOnErrorHandler(method)
     return function(error)
-        display_error(("%s (Error %d, %s)"):format(error.message, error.code, method))
+        display_error(("%s (Error %s, %s)"):format(
+            error.message, LSPErrorCodes[error.code], method
```

Comment:
We should probably fall back to `error.code` if the code is not in `LSPErrorCodes`.